language/move-stdlib/tests/BasicTests.move

```diff
@@ -0,0 +1,18 @@
+#[test_only]
```

Comment:
We try to keep the naming convention in the tests the the tests for the module in  `X.move` is named `XTests.move`. Could you please rename this to `ErrorsTests.move`?